client/src/components/Header/BoardTitle.jsx

```diff
@@ -0,0 +1,9 @@
+import styles from './styles.module.scss'
+
+const BoardTitle = () => {
+  return (
+    <div className={styles.boardTitle}>Platform Launch</div>
```

Comment:
❓  it seems that when we toggle to dark theme, the text is not updating accordingly. We can do so in a follow-up PR so we don't block this one, or here. Up to you! ;)  This is what I see: 
![Screenshot 2024-07-23 at 1 35 35 PM](https://github.com/user-attachments/assets/ebb18263-e219-4884-b3b9-b912afa4612d)

Reply:
Also. we might need to remove the padding or margin for this component when dark. W/o looking too deep into this I'm not sure that would fix that like in between the components.

Reply:
I made a new task to apply dark theme to the header.